scripts/build.sh

```diff
@@ -18,7 +18,7 @@ GO_BUILD_ENV=("CGO_ENABLED=0" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS}" "GOOS=${GOOS}"
 toggle_failpoints() {
   mode="$1"
   if command -v gofail >/dev/null 2>&1; then
-    run gofail "$mode" server/etcdserver/ server/mvcc/backend/
+    run gofail "$mode" server/etcdserver/ server/storage/backend/
```

Comment:
The change looks good. 

But the directory change was made more than half year ago in this [commit](https://github.com/etcd-io/etcd/commit/5e40a8b00cc1be3ce0dd3d7e6947d417253fb03f), why we do not see any issue in such a long time? cc @serathius @ptabor @spzala